training/culture-and-process/CULTURE.md

```diff
@@ -0,0 +1,48 @@
+Culture in DP
+===========================
+Our culture and how we work is important for us to create an inclusive, empathetic and successful environment that works for everybody. 
+
+
+### Principles
+
+#### Inclusion
+We believe people from different backgrounds, with different identities and experiences, make our product, and our team, better. 
```

Comment:
Could probably remove "with different identities" as I think backgrounds/experiences broadly covers the same space

"We believe people from different backgrounds and experiences make our products and our team better"

Reply:
I'm not sure. I think someone's identity or at least how they choose to identify can be different to their background?

Reply:
Yeah, was thinking about this. I think it makes sense to leave as is